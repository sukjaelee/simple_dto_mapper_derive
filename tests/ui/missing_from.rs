use simple_dto_mapper_derive::DtoFrom;

mod types {
    pub struct Source {
        pub id: String,
    }
}

#[derive(DtoFrom)]
struct Dto {
    id: String,
}

fn main() {}
