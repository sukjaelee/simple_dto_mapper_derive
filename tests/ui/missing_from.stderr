error: Expected `#[dto(from = Type)]` on the struct.
 --> tests/ui/missing_from.rs:9:10
  |
9 | #[derive(DtoFrom)]
  |          ^^^^^^^
  |
  = note: this error originates in the derive macro `DtoFrom` (in Nightly builds, run with -Z macro-backtrace for more info)
